lib/engine/game/g_1894/step/special_buy.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/special_buy'
+
+module Engine
+  module Game
+    module G1894
+      module Step
+        class SpecialBuy < Engine::Step::SpecialBuy
+          attr_reader :ferry_marker
+
+          def buyable_items(entity)
+            return [@ferry_marker] if @game.loading || @game.can_buy_ferry_marker?(entity)
```

Comment:
what's going on here

Reply:
It's the same thing as coal marker in 1828. Here you give the possibility to buy it if the corporation is connected to England.